core/src/main/java/org/mifos/mobilewallet/core/data/fineract/repository/FineractRepository.java

```diff
@@ -240,7 +240,8 @@ public ResponseBody call(ResponseBody responseBody) {
     }
 
     public Observable<GenericResponse> updateStandingInstruction(long standingInstructionId,
-                                                                 StandingInstructionPayload standingInstructionPayload) {
+                                                                 StandingInstructionPayload
+                                                                         standingInstructionPayload) {
```

Comment:
This doesn't seem good, just rename the variable to `payload` not use a new line for just the variable and it will probably fit in 100 characters

Reply:
![image](https://user-images.githubusercontent.com/66490629/162464007-5f220722-2d1b-4a04-b257-95ca40ee0780.png)

![image](https://user-images.githubusercontent.com/66490629/162464114-b2027297-9325-4fcf-b735-2f77a08f5eed.png)

Reply:
renaming the variable to `payload` arises same error

Reply:
Just rename the variable to data. Btw you can use any name you want which avoids this error

Reply:
Okay